plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageProcessor.java

```diff
@@ -2956,6 +3033,61 @@ public void handleTargets(boolean add, ModifyTargetsCommand.TargetTypeToRemove t
         }
     }
 
+    private String waitForDatastoreName(HostDatastoreSystemMO hostDatastoreSystemMO, String iqn) throws Exception {
+        long secondsToWait = 120;
```

Comment:
Are you planning to externalize this variable?

Reply:
I hadn't planned to. In almost every case, the datastore will mount in under 30 seconds. 120 is really a super worst-case scenario. If we have waited that long and not see the datastore mounted yet, just fail. The normal case is the datastore will mount in 5 - 10 seconds and we won't get anywhere near 120 seconds.

Reply:
One way I could make it more clear that it's a constant is to declare the variable final. Would you prefer that I add that keyword to it?

Reply:
I think it is just fine as is.

Reply:
just to butt in here; it is used at line 2661 as well. Is there any reason those two variables with the same name would have different values? I suspect not and if I'm right I'd like to see a constant.

Reply:
Done